### DISCUSSION TITLE
Add cadence and resistance sensors for a very affordable complete solution?

### DISCUSSION THREAD

**Discussion by chinasaur** [doudar/SmartSpin2k#224] [Ideas]
I wonder if there's interest in this. I imagine this group has ideas on a cheap but decent, readily available spin bike that would be easy to add cadence and resistance sensors to in addition to the resistance actuator. Cadence is pretty trivial, resistance I'm not sure. The OEM resistance sensor on my C6 is not so great, but basically gets the job done.

Or do others think that off-shelf power pedals is a better solution? I've never owned power pedals, but IIUC they are more accurate than a bike resistance sensor, but relatively expensive. Perhaps bike resistance gets the job done?

**Comment by tootechy** [comment]
I put the tempo power meter on my IC4, works great for much less than the pedals: https://www.tempopowermeters.com/product-page/spin-bike-power-meter

**Reply by chinasaur** [reply]
Looks really nice; I might try this since my C6 resistance has never been well calibrated, and recently my resistance sensor seems to have died. (Not related to my hacking on the console I don't think, although more debugging to come.)

I chatted with Tempo and it sounds like they can install on the "dumb" IC3 as well, which is a lot cheaper. So that already looks like an attractive new build option.
